### PR TITLE
Added new tutorials to navigation

### PR DESCRIPTION
**Pull request by thomelane**
Added to toctree. Added cards. A few other minor edits to tutorials to remove warning in build logs.

### CODE REVIEW

**Comment by thomelane** [comment]
@vishaalkapoor can you do the same for your tutorials, specifically add to toctree and add cards in the relevant places on the site, thanks!

**Comment by vishaalkapoor** [comment]
> @vishaalkapoor can you do the same for your tutorials, specifically add to toctree and add cards in the relevant places on the site, thanks!

@thomelane I have three tutorials (nn, init, custom loss) and they all fall under the gluon card, so should we be adding them here? Also, I believe they are already in all indices. Do you see any places where they are missing?

File: python/guide/packages/gluon/activations/activations.md [hunk 1]
```diff
@@ -50,7 +51,7 @@ visualize_activation(mx.gluon.nn.Activation('sigmoid'))
 ```
 
 
-![png](images/sigmoid.png)
+![](images/sigmoid.png)
```
Comment (vishaalkapoor) [inline python/guide/packages/gluon/activations/activations.md]: Can we add a descriptive description? A plot of the sigmoid function?

Comment (vishaalkapoor) [reply python/guide/packages/gluon/activations/activations.md]: Same in other occurences

File: python/guide/packages/gluon/custom-loss/custom-loss.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
 
-# Custom Loss functions in Gluon
+# Custom Loss Blocks
```
Comment (vishaalkapoor) [inline python/guide/packages/gluon/custom-loss/custom-loss.md]: Blocks for Custom Loss would be better wording

File: python/guide/packages/gluon/custom-loss/custom-loss.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
+A critical component of training neural networks is the loss function. A loss function is a quantative measure of how bad the predictions of the network are when compared to ground truth labels. Given this score, a network can improve by iteratively updating its weights to minimise this loss. Some tasks use a combination of multiple loss functions, but often you'll just use one. MXNet Gluon provides a number of the most commonly used loss functions, and you'll choose certain functions depending on your network and task. Some common task and loss function pairs include:
```
Comment (vishaalkapoor) [inline python/guide/packages/gluon/custom-loss/custom-loss.md]: I would reverse this.
Loss functions are crucial/integral to traiing neural networks.

File: python/guide/packages/gluon/index.rst [hunk 4]
```diff
@@ -134,7 +152,10 @@ Applications Topics
    data
    image-augmentation
    nn
```
Comment (vishaalkapoor) [inline python/guide/packages/gluon/index.rst]: nn, init, custom-layers are here.

**Comment by vishaalkapoor** [comment]
Some comments, LGTM.

**Comment by thomelane** [comment]
thanks for the review @vishaalkapoor. and thanks for confirming your tutorials are already in the navigation and are cards. sorry for not looking beforehand!

File: python/guide/packages/gluon/activations/activations.md (hunk 1 above)
Comment (thomelane) [reply python/guide/packages/gluon/activations/activations.md]: Will add this to the QA checklist. Will remove undescriptive 'png' for now, and ask tutorial owners to add to their tutorials as required, as I don't think this has been done anywhere.

File: python/guide/packages/gluon/custom-loss/custom-loss.md (hunk 2 above)
Comment (thomelane) [reply python/guide/packages/gluon/custom-loss/custom-loss.md]: Could make it harder to find in the navigation I think. Starting with Custom Loss cuts to the chase.

File: python/guide/packages/gluon/index.rst (hunk 4 above)
Comment (thomelane) [reply python/guide/packages/gluon/index.rst]: thanks for confirming!

File: python/guide/packages/gluon/custom-loss/custom-loss.md (hunk 3 above)
Comment (thomelane) [reply python/guide/packages/gluon/custom-loss/custom-loss.md]: Reworded sentence.
